rotkehlchen/tests/unit/test_accounting.py

```diff
@@ -918,6 +921,52 @@ def test_fees_count_in_cost_basis(accountant):
     assert errors == [error]
 
 
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_fees_in_received_asset(accountant):
+    """
+    Test the sell trade where the fee is nominated in the asset received. We had a bug
+    where the PnL report said that there was no documented adquisition.
+    """
+    history = [
+        {
+            'timestamp': 1609537953,
+            'base_asset': 'ETH',
+            'quote_asset': A_USDT.identifier,
+            'trade_type': 'sell',
+            'rate': 1000,
+            'fee': '0.10',
+            'fee_currency': A_USDT.identifier,
+            'amount': 0.02,
+            'location': 'binance',
+        },
+    ]
+    ledger_actions_list = [
+        LedgerAction(
+            identifier=0,
+            timestamp=Timestamp(1539713238),
+            action_type=LedgerActionType.INCOME,
+            location=Location.BINANCE,
+            amount=FVal(1),
+            asset=A_ETH,
+            rate=None,
+            rate_asset=None,
+            link=None,
+            notes='',
+        ),
+    ]
+    accounting_history_process(
+        accountant,
+        start_ts=1539713238,
+        end_ts=1624395187,
+        history_list=history,
+        ledger_actions_list=ledger_actions_list,
+    )
+    warnings = accountant.msg_aggregator.consume_warnings()
+    assert len(warnings) == 0
+    errors = accountant.msg_aggregator.consume_errors()
+    assert len(errors) == 0
```

Comment:
please also assert here for the general P/L calculation to make sure it is correct.